src/remote.ts

```diff
@@ -156,6 +181,9 @@ export class Remote {
 
     const workspaceName = `${parts.username}/${parts.workspace}`
 
+    // Migrate "session_token" file to "session", if needed.
+    this.storage.migrateSessionToken(parts.label)
```

Comment:
We should await this to make sure the next line is not reading from a file that does not exist yet

```suggestion
    await this.storage.migrateSessionToken(parts.label)
```